src/core/public/chrome/ui/header/collapsible_nav_group_enabled.scss

```diff
@@ -2,8 +2,8 @@
   border: none !important;
 
   .nav-link-item {
-    padding: $ouiSize / 4 $ouiSize;
-    border-radius: $ouiSize;
+    padding: calc($euiSize / 4) $euiSize;
+    border-radius: $euiSize;
```

Comment:
Are these changes expected? Seems unrelated

Reply:
It is expected, as Miki told me to do so.